assets/js/modules/analytics/common/account-create.js

```diff
@@ -85,6 +87,10 @@ export default function AccountCreate() {
 		[ createAccount, setIsNavigating ]
 	);
 
+	// If the user clicks "Back", rollback settings to restore saved values, if any.
+	const { rollbackSettings } = useDispatch( STORE_NAME );
+	const handleBack = useCallback( () => rollbackSettings() );
```

Comment:
Why do we need to rollback settings here?

Reply:
The minimum action needed is to change the `accountID` to not be `account_create`.

I thought it made sense to use the rollback for this as otherwise all previous selections would be lost as they are cleared when selecting `account_create`. This only really makes a difference on the settings view of course, but slightly better than just clearing the `accountID` I think.